test/integration/scan_test.go

```diff
@@ -670,3 +670,21 @@ func TestRunKicsScanWithAdditionalParams(t *testing.T) {
 
 	assert.Assert(t, outputBuffer != nil, "Scan must complete successfully")
 }
+
+func TestScanCreateWithAPIKeyNoTenant(t *testing.T) {
```

Comment:
Maybe here we can launch the scan with async, or do some other command instead of launching a scan... We just need to test if the auth works, no need to wait for a full scan.